tests/common.py

```diff
@@ -148,25 +148,44 @@ def prepare_reference_nonbonded(params, exclusion_idxs, scales, lambda_plane_idx
     return ref_total_energy
 
 
-def prepare_water_system(x, lambda_plane_idxs, lambda_offset_idxs, p_scale, cutoff):
+def prepare_system_params(x: NDArray, sigma_scale: float = 5.0) -> NDArray:
+    """
+    Prepares random parameters given a set of coordinates. The parameters are adjusted to be the correct
+    order of magnitude.
+
+    Parameters
+    ----------
+
+    x: Numpy array of Coordinates
 
+    sigma_scale: Factor to scale down sigma values by
+    """
     assert x.ndim == 2
     N = x.shape[0]
-    # D = x.shape[1]
-
-    assert N % 3 == 0
 
     params = np.stack(
```

Comment:
I noticed this same code in a few places and wanted to unify it. Though some places had different sigma_scale values, which didn't seem important (besides not setting it to 1) so I defaulted to 5.0